### DISCUSSION TITLE
请问一下大家，Alist复制文件遇到相同文件名是什么策略？跳过还是覆盖？

### DISCUSSION THREAD

**Discussion by djhdj1** [AlistGo/alist#6382] [Q&A]
请问一下大家，Alist复制文件遇到相同文件名是什么策略？跳过还是覆盖？有没有可以配置的地方

**Comment by pongfcnkl** [comment]
看网盘，部分网盘覆盖，部分网盘是重命名
可以自己改源码把NoOverwriteUpload关闭

**Reply by djhdj1** [reply]
你说的这个设置好像只有部分网盘有啊，没有的比如Onedrive就是直接跳过？

**Reply by pongfcnkl** [reply]
@djhdj1 
没有的就自己加上试试
false是开启覆盖，你可以用rclone copyto配合来跳过
搜issuse，115重复那个有我的命令，可以借鉴

**Comment by djhdj1** [comment]
> 看网盘，部分网盘覆盖，部分网盘是重命名 可以自己改源码把NoOverwriteUpload关闭

还有个问题就改成false就是同名文件跳过？
